src/System.Private.Xml/src/System/Xml/Dom/XmlDocument.cs

```diff
@@ -22,7 +22,7 @@ public class XmlDocument : XmlNode
         private DomNameTable _domNameTable; // hash table of XmlName
         private XmlLinkedNode _lastChild;
         private XmlNamedNodeMap _entities;
-		// TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
+        // TODO: _htElementIdMap can be Dictionary<string, List<WeakReference<XmlElement>>>
```

Comment:
I looked through the others and none seem worth issues. I assume most carried over from years ago.

---
In reply to: [110315858](https://github.com/dotnet/corefx/pull/18042#discussion_r110315858) [](ancestors = 110315858,110308973)